pkgs/top-level/all-packages.nix

```diff
@@ -3718,7 +3720,7 @@ with pkgs;
   lzip = callPackage ../tools/compression/lzip { };
 
   luxcorerender = callPackage ../tools/graphics/luxcorerender { };
-  
+
```

Comment:
While I hate trailing whitespace could you remove these changes from this PR?

Reply:
@adisbladis done, PTAL.